virtual_rainforest/models/hydrology/constants.py

```diff
@@ -75,9 +70,25 @@ class HydroConsts:
     heat_transfer_coefficient: float = 12.5
     """Heat transfer coefficient, :cite:p:`van_de_griend_bare_1994` """
 
-    flux_to_mm_conversion: float = 3.35e-4
-    """Factor to convert evaporative flux to mm."""
-
     stream_flow_capacity: float = 5000.0
     """Stream flow capacity, mm per timestep. This is curretly an arbitrary value, but
     could be used in the future to flag flood events."""
+
+    intercept_param_1: float = 0.935
+    """Parameter in equation that estimates maximum canopy interception capacity after
+    Von Hoyningen-Huene, (1981)."""
+
+    intercept_param_2: float = 0.498
+    """Parameter in equation that estimates maximum canopy interception capacity after
+    Von Hoyningen-Huene, (1981)."""
+
+    intercept_param_3: float = 0.00575
+    """Parameter in equation that estimates maximum canopy interception capacity after
+    Von Hoyningen-Huene, (1981)."""
+
+    veg_density_param: float = 0.046
+    """Parameter used to estimate vegetation density for maximum canopy interception
+    capacity estimate after Von Hoyningen-Huene, (1981)."""
+
+    groundwater_capacity: float = 0.9
+    """Ground water storage capacity."""
```

Comment:
Does this have a unit?

Reply:
At the moment, this is relative volumetric water content to match the soil water capacity. This might be replaced with the implementation of below ground horizontal flow.